src/platform/site-wide/sass/modules/_m-homepage.scss

```diff
@@ -182,23 +182,6 @@
     }
   }
 
-  &-icon {
-    font-size: 1.5em;
-    display: inline-block;
-    position: relative;
-    &::before {
-      position: absolute;
-      margin: 0;
-      top: 15px;
-      left: -7.75px;
-      font-family: "Font Awesome 5 Free"
-    }
-    &.fa-user-circle {
```

Comment:
This is still used in src/applications/static-pages/widget-creators

Reply:
It looks like this whole block is used there, maybe

Reply:
@randimays 
I see this:
`  const homePageStoreListener = () => {
    if (root && isLoggedIn(store.getState())) {
      root.innerHTML = `<a href="${dashboardUrl}" class="homepage-button">
        <div class="icon-wrapper"><va-icon icon="account_circle" size="3" class="homepage-button-icon" /></div>
        <div class="button-inner"><span>Go to your personalized “My VA” page</span></div>
      </a>`;
      unsubscribe();
    }`
    
    But now I wonder, should we eliminate the class name? va-icon size is controlled internally, am not sure why  we would need the `display:nline-block` and the rest is styling the old fa-icon which is no longer present. Suggestions?

Reply:
Oh good call, thanks Ray! My branch was out of date; I see the `<va-icon>` now.

Hm, I see that we're actually using `.homepage-button` in spots in content-build. It's a known issue that content-build still references some styles in vets-website since the repos used to coexist. We can't remove the whole class block unless we fix that.